src/utils/math.helper.ts

```diff
@@ -0,0 +1,4 @@
+export function isNumber(value: unknown) {
+  if (typeof value === "number") return true

```

Comment:
The function is correctly checking if the input type is a number, but consider using a clearer, more conventional format. You could utilize an implicit return for better readability, like this: `return typeof value === 'number';`.